package.json

```diff
@@ -104,6 +104,6 @@
     "typescript": "^5.3.3"
   },
   "dependencies": {
-    "markdown-it-directive": "^2.0.2"
+    "@diplodoc/directive": "^0.1.0"
```

Comment:
lets update with https://github.com/hilookas/markdown-it-directive/pull/9#issuecomment-2472907718

Reply:
done

Reply:
this is not necessary